tests/test_statics.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from falcon.request import Request
+from falcon.response import Response
+from falcon.statics import Statics
+import falcon.testing as testing
+
+
```

Comment:
This test module looks like a good start. Once you have the unit tests fleshed out, it would be great if you could also add a few functional tests ala `test_sinks.py` just to make sure everything is working end-to-end. Along the way, it may make sense to allow overriding the statics type (similar to ```API._request_type```). This would make testing easier while also providing a way for developers to customize the behavior.